src/object/srv_obj_migrate.c

```diff
@@ -805,9 +982,11 @@ migrate_fetch_update_parity(struct migrate_one *mrone, daos_handle_t oh,
 		DP_UOID(mrone->mo_oid), mrone, DP_KEY(&mrone->mo_dkey),
 		mrone->mo_iod_num, mrone->mo_epoch);
 
-	rc = dsc_obj_fetch(oh, mrone->mo_epoch, &mrone->mo_dkey,
-			   mrone->mo_iod_num, mrone->mo_iods, sgls, NULL,
-			   DIOF_TO_LEADER | DIOF_FOR_MIGRATION, NULL);
+	rc = daos_iov_alloc(&csum_iov_fetch, CSUM_BUF_SIZE, false);
+	if (rc)
+		D_GOTO(out, rc);
+	rc = mrone_obj_fetch(mrone, oh, sgls, &csum_iov_fetch);
+	tmp_csum_iov_fetch = csum_iov_fetch;
```

Comment:
same here, you probably do not need fetch csum explicitly for parity rebuild.

Reply:
Yes, this will need to be fixed. but there are failing EC tests (w/ checksums) that make it difficult to verify a change here. I created a new story to resolve this so these changes can land for 1.2. EC + checksums aren't required for 1.2.
DAOS-6811 is the new story.